Bottom: 08871e7423ca54f37a5a657ef415703be8050119
Top:    421b2b8cb99a21c30ba812290ac75b27b8de1c2f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-02-13 18:44:28 +0000

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 4edb048..99d243a 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2460,7 +2460,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
